services/distribution/src/test/java/app/coronawarn/server/services/distribution/common/Helpers.java

```diff
@@ -1,35 +1,29 @@
 package app.coronawarn.server.services.distribution.common;
 
-import static app.coronawarn.server.services.distribution.assembly.appconfig.YamlLoader.loadYamlIntoProtobufBuilder;
-import static java.io.File.separator;
```

Comment:
@FelixRottler please check your IDE settings for `import` organization... as per checkstyle rules, `statics` should be at the top